dbscripts/db-init.sql

```diff
@@ -202,6 +202,7 @@ CREATE TABLE LoanPayments (
 -- ProductPayments
 
 CREATE TABLE ProductPayments (
+	productPaymentUuid UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
```

Comment:
what's the difference between this and productTransactionUuid?

Reply:
this is a unique identifier on this table. productTransactionUuid relates it to one product transaction.